lib/logger.js

```diff
@@ -2,12 +2,43 @@
 
 const deprecate = require('depd')('propsd');
 const Winston = require('winston');
+const expressWinston = require('express-winston');
 
+/**
+ * Create a logger instance
+ * @param {string} level
+ * @param {string} filename
+ * @returns {Winston.Logger}
+ * @constructor
+ */
 function Logger(level, filename) {
-  const logLevel = level || 'info';
+  const logLevel = level.toUpperCase() || 'INFO';
+
+  /* eslint-disable rapid7/static-magic-numbers */
+  const javaLogLevels = {
```

Comment:
Yeah, [Logback](http://logback.qos.ch/apidocs/ch/qos/logback/classic/Level.html) uses (not listed in order of severity) `DEBUG`, `ERROR`, `INFO`, `WARN`, AND `TRACE`. The problem is that `SILLY` and `VERBOSE` are used for two different things but map more closely to `TRACE`.

Agreed that the var should be renamed. Maybe `javaLikeLogLevels`? 😝